### ISSUE TITLE
Loading class 'com.mysql.jdbc.Driver'. This is deprecated. The new driver class is 'com.mysql.cj.jdbc.Driver'. The driver is automatically registered via the SPI and manual loading of the driver class is generally unnecessary.

### ISSUE THREAD

**Issue by mibby** [Realizedd/TokenManager#50]
```
[23:24:27] [Server thread/INFO]: [TokenManager] Enabling TokenManager v3.2.6-SNAPSHOT
[23:24:27] [Server thread/INFO]: [TokenManager] Loaded Config.
[23:24:27] [Server thread/INFO]: [TokenManager] Loaded Lang.
[23:24:27] [Server thread/INFO]: [TokenManager] ===============================================
[23:24:27] [Server thread/INFO]: [TokenManager] TokenManager has detected your server as online mode.
[23:24:27] [Server thread/INFO]: [TokenManager] DataManager will operate with UUIDs.
[23:24:27] [Server thread/INFO]: [TokenManager] If your server is NOT in online mode, please manually set online-mode in TokenManager's config.yml.
[23:24:27] [Server thread/INFO]: [TokenManager] ===============================================
[23:24:27] [Server thread/WARN]: Loading class `com.mysql.jdbc.Driver'. This is deprecated. The new driver class is `com.mysql.cj.jdbc.Driver'. The driver is automatically registered via the SPI and manual loading of the driver class is generally unnecessary.
[23:24:27] [Server thread/INFO]: [TokenManager] Loaded DataManager.
[23:24:27] [Server thread/INFO]: [TokenManager] Loaded ShopConfig.
[23:24:27] [Server thread/INFO]: [TokenManager] Loaded ShopManager.
[23:24:27] [Server thread/INFO]: [TokenManager] Loaded WorthConfig.
```

**Comment by mibby** [comment]
I assume a proper fix would be to try a newer mysql driver first? Both logblock & a lwc fork I use fixed their warnings with;

```
            try {
                Class.forName("com.mysql.cj.jdbc.Driver");
            } catch (ClassNotFoundException ignored) {
                Class.forName("com.mysql.jdbc.Driver");
            }
```

**Comment by broken1arrow** [comment]
> 
> 
> I assume a proper fix would be to try a newer mysql driver first? Both logblock & a lwc fork I use fixed their warnings with;
> 
> ```
>             try {
>                 Class.forName("com.mysql.cj.jdbc.Driver");
>             } catch (ClassNotFoundException ignored) {
>                 Class.forName("com.mysql.jdbc.Driver");
>             }
> ```

You don't even need this any more.

**Comment by Realizedd** [comment]
Fixed in bf887c1
